lib/Dialect/FIRRTL/Transforms/LowerXMR.cpp

```diff
@@ -199,6 +197,21 @@ class LowerXMRPass : public LowerXMRBase<LowerXMRPass> {
 
   // Replace the RefResolveOp with verbatim op representing the XMR.
   LogicalResult handleRefResolve(RefResolveOp resolve) {
+    auto resWidth = getBitWidth(resolve.getType());
+    if (resWidth.has_value() && resWidth.value() == 0) {
+      // Donot emit 0 width XMRs, instead rely on `LowerToHW` to remove these
+      // temporary 0-width wires.
+      ImplicitLocOpBuilder builder(resolve.getLoc(), resolve);
+      auto dummyWire = builder.create<WireOp>(resolve.getType());
+      resolve.getResult().replaceAllUsesWith(dummyWire);
```

Comment:
Maybe I'm missing something, do we even need `dummyWire`? Can we directly use aggregate constant?

Reply:
I was considering there could be an assignment to the wire , But I think you are right, the wire is not required.